huaweicloud/services/evs/data_source_huaweicloud_evs_volumes.go

```diff
@@ -0,0 +1,231 @@
+package evs
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/chnsz/golangsdk/openstack/evs/v2/cloudvolumes"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/helper/hashcode"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
+)
+
+func DataSourceEvsVolumesV2() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceEvsVolumesV2Read,
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"availability_zone": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"dedicated_storage_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"dedicated_storage_name": {
```

Comment:
`dedicated_storage_id` and `dedicated_storage_name`  are not described in the document, but they are useful in read.
What are these two arguments for?

Reply:
All parameters related to dedicated storage have been deleted.